taps/executor/parsl.py

```diff
@@ -180,7 +185,7 @@ class HTExConfig(BaseModel):
         description='Ports used by Parsl to connect to interchange.',
     )
 
-    def get_executor(self) -> HighThroughputExecutor:
+    def get_executor(self, label: str = 'taps-htex') -> HighThroughputExecutor:
```

Comment:
I think we should just make `label` a field of the class. Then we won't need to modify the signature of `get_executor()`.
```python
label: str = Field('taps-htex', description='Executor label.')
```